### PR TITLE
added documentation of features in Cake for Rider version 1.0.0

### PR DESCRIPTION
**Pull request by nils-a**

### CODE REVIEW

File: input/docs/integrations/editors/rider/templates.md [hunk 1]
```diff
@@ -1,13 +1,67 @@
...
+* `cake-tool-full`
+  * Provides a more complete tool pre-processor directive, where source, package name and version can be changed
+  * _Default Value:_ `#tool "nuget:https://www.nuget.org/api/v2?package=Cake.Foo&version=1.2.3"`
+* `setup`
+  * Provides a sample setup definition
+* `task-simple`
+  * Provides a basic task definition, where the name of the task can be changed
+  * _Default Value:_ `Task("name");`
+* `task-action`
+  * Provides a more complex task definition, including an `.Does` body, where the name of the task can be changed
```
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:65]: **nitpick**: shouldn't it be "including _a_ `.Does` body"?

File: input/docs/integrations/editors/rider/templates.md [hunk 2]
```diff
@@ -1,13 +1,67 @@
...
+![Run configurations](/assets/img/cake-rider/docs/item-templates.png){.img-responsive}
+
+# Live templates (sometimes called code snippets)
+
+Live templates are only available in Cake files.
+
+Available live templates:
+
+* `cake-addin`
+  * Provides a basic addin pre-processor directive, where the package name and version can be changed
```
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:31]: **suggestion**: I would format the word "_addin_" as `#addin`.

File: input/docs/integrations/editors/rider/templates.md [hunk 3]
```diff
@@ -1,13 +1,67 @@
...
+  * _Default Value:_ `#addin "nuget:?package=Cake.Foo&version=1.2.3"`
+* `cake-addin-full`
+  * Provides a more complete addin pre-processor directive, where source, package name and version can be changed
+  * _Default Value:_ `#addin "nuget:https://www.nuget.org/api/v2?package=Cake.Foo&version=1.2.3"`
+* `cake-argument`
+  * Provides code for basic input argument parsing, where variable name, argument name and default value can be changed
+  * _Default Value:_ `var argumentName = Argument("Argument name", "Default value");`
+* `cake-load`
+  * Provides a basic load pre-processor directive, where the path to the .cake file can be changed
```
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:40]: **suggestion**: I would format the word "_load_" as `#load`.

File: input/docs/integrations/editors/rider/templates.md [hunk 4]
```diff
@@ -1,13 +1,67 @@
...
+  * _Default Value:_ `#load "scripts/utilities.cake"`
+* `cake-load-nuget`
+  * Provides a more complex load pre-processor directive, where package name and version can be changed
+  * _Default Value:_ `#load "nuget:?package=Cake.Foo&version=1.2.3"`
+* `cake-load-nuget-full`
+  * Provides a more complex load pre-processor directive, where source, package name and version can be changed
+  * _Default Value:_ `#load "nuget:https://www.nuget.org/api/v2?package=Cake.Foo&version=1.2.3"`
+* `cake-reference`
+  * Provides a basic reference pre-processor directive, where path to the assembly can be changed
```
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:49]: **suggestion**: I would format the word "_reference_" as `#reference`.

File: input/docs/integrations/editors/rider/templates.md [hunk 5]
```diff
@@ -1,13 +1,67 @@
...
+  * _Default Value:_ `#reference "bin/myassembly.dll"`
+* `cake-sample`
+  * Provides a complete sample build Cake script including setup and teardown actions, a single task, and argument parsing
+* `cake-tool`
+  * Provides a basic tool pre-processor directive, where the package name and version can be changed
```
Comment (ecampidoglio) [inline input/docs/integrations/editors/rider/templates.md:54]: **suggestion**: I would format the word "_tool_" as `#tool`.

**Review by ecampidoglio — APPROVED** [review]
LGTM with a few suggestions.

**Comment by nils-a** [comment]
Cake for Rider 1.0.0 has been released 🎉 
I will merge this PR, once it has been approved by JetBrains

**Review by gep13 — APPROVED** [review]
LGTM!

**Review by augustoproiete — APPROVED** [review]
LGTM!

image

**Pull request merged**